### PR TITLE
6: Fix hero component and create new pages

### LINKED ISSUE #923: Create Hero component
### What's your idea?

Redesign the hero component in acmcsuf.com following the provided image:

Light mode desktop | Light mode mobile | Dark mode desktop | Dark mode mobile
---|---|---|---
light-desktop-hero | light-mobile-hero | dark-desktop-hero | dark-mobile-hero

### PR DESCRIPTION
**Pull request by k4rni**
- Add separate `.svelte` files per component
- Create hero component 
- Adjust hero component in desktop mode to fit to figma wireframe

Fix: #923.

### CODE REVIEW

**Comment by k4rni** [comment]
Please note in order to see the actual changes in the preview or in dev mode, it is on the `/6` page.

cc: @tomasohCHOM

**Comment by tomasohCHOM** [comment]
I was able to make those changes by making the text within the h1 all span elements to make it have consistent behavior. Added some gap within grid elements in desktop view too.

File: src/routes/(site)/6/hero.svelte [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+
+    <img
+      src="/assets/png/capy-power.png"
+      alt="Chip the Capybara surrounded by all ACM Team badges"
+    />
+  </div>
+</section>
+
+<style>
+  section {
```
Comment (k4rni) [inline src/routes/(site)/6/hero.svelte:25]: Move `` above text container in mobile view

File: src/routes/(site)/6/hero.svelte [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+    display: grid;
+    justify-self: center;
+  }
+
+  section .hero-inner-container img {
+    max-width: clamp(20rem, 17.342rem + 10.13vw, 30rem);
+    justify-self: center;
+  }
+
+  @media screen and (min-width: 768px) {
```
Comment (k4rni) [inline src/routes/(site)/6/hero.svelte:70]: Replace `@media` queries with container queries

**Review by EthanThatOneKid — APPROVED** [review]
LGTM! Thanks for kicking off the 6.0 redesign, @karnikaavelumani and @tomasohCHOM!

**Comment by EthanThatOneKid** [comment]
Let’s [convert the unresolved comments into issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/creating-an-issue#creating-an-issue-from-a-comment) in order to make sure they get followed up on.

**Pull request merged**